pkg/controller/volumeattachment.go

```diff
@@ -50,9 +49,10 @@ type ReconcileVolumeAttachment struct {
 	azVolumeClient azVolumeClientSet.Interface
 	kubeClient     kubeClientSet.Interface
 	// retryMap allows volumeAttachment controller to retry Get operation for AzVolumeAttachment in case the CRI has not been created yet
-	retryMap   map[string]*uint32
-	retryMutex sync.RWMutex
-	namespace  string
+	retryMap *SharedMap
```

Comment:
Why share per-PV retries between the `PV` and `VolumeAttachment` controllers? Shouldn't their retries be independent of each other?

Reply:
It is not shared between the two controller. This might have been misleading due to name of the type `SharedMap`.

Reply:
I have replaced all `SharedMap` usages with `sync.Map` by the way.

Reply:
> It is not shared between the two controller. This might have been misleading due to name of the type `SharedMap`.

Yeah, I was confused. Thanks for the clarification.